src/platforms/ctxlink/WiFi_Server.c

```diff
@@ -1244,7 +1395,16 @@ void app_task(void)
 	}
 	case app_state_wait_for_server: {
 		if (is_gdb_server_running())
```

Comment:
This `if` becomes multi-line with the inner `if` addition - please either add parens, or.. due to the construction of the inner.. combine as in `if (is_gdb_server_running() && is_ip_address_assigned())`.

Reply:
Fixed.